src/translations/en.json

```diff
@@ -2240,6 +2240,10 @@
             "create": "Create Group",
             "creating_group": "Creating Group"
           },
+          "visualization": {
+            "header": "Zigbee Home Automation - Visualization",
```

Comment:
I don't think we need the `Zigbee Home Automation - `?

Network Visualization?
Network Map?

Reply:
Network Visualization

Reply:
Done